src/common/components/atoms/card.tsx

```diff
@@ -12,6 +12,9 @@ const Card = React.forwardRef<
       "rounded-xl border bg-card text-card-foreground",
       className,
     )}
+    style={{
+      boxShadow: "var(--user-theme-fidget-shadow)",
```

Comment:
Because this Card has `overflow-hidden`, we need fidget shadow to be here

Reply:
Not sure we want fidget shadow to affect cards globally. Might be better to apply this style to the Card component in FidgetWrapper

Reply:
That makes sense! Updated. Thank you!